### ISSUE TITLE
Inline JavaScript

### ISSUE THREAD

**Issue by bennypowers** [alibaba/structure-view#2]
Nice plugin!

Would love to have structure of inline javascript

```html
<dom-module id="my-element">
  <template>
    <!-- etc -->
  </template>
  <script>
  // structure me!
  Polymer({
    is: 'my-element',
    method: function() {},
    _private: () => true,
  });
  </script>
</dom-module>
```

Great work!

**Comment by Joouis** [comment]
@bennypowers Thank you. The structure of inline javascript is in the plan.

**Comment by Joouis** [comment]
This issue is fixed by commits 7a23680, e8f728a, b6e00a9.
